app/views/users/show.html.erb

```diff
@@ -1,161 +1,240 @@
+      <% else %>
+        <%= raw(sanitize(@user.profile, scrubber: scrubber)) %>
+      <% end %>
+      </div>
+
+      <div class="button-list h-p-2">
+        <% if user_signed_in? %>
           <%= link_to new_subscription_path(type: 'user', qualifier: @user.id, return_to: request.path), class: "button is-outlined is-small" do %>
             <i class="fas fa-envelope"></i> Subscribe to user
           <% end %>
-        </div>
-      <% end %>
-      <% if current_user&.is_moderator %>
-        <div>
+        <% end %>
+        <% if current_user&.is_moderator %>
           <a href="<%= mod_user_path(@user) %>" class="button is-danger is-outlined is-small" data-drop="#mod-tools-drop"><i class="fas fa-shield-alt"></i> Moderator Tools</a>
-
           <div class="droppanel" id="mod-tools-drop">
             <div class="droppanel--header">quick actions</div>
             <div class="droppanel--menu">
               <a href="/users/<%= @user.id %>/mod/activity-log">full activity log</a>
               <a href="/mod/votes/user/<%= @user.id %>">suspicious voting patterns</a>
+              <a href="/users/<%= @user.id %>/mod/annotations">annotations on user</a>
+              <a href="/users/<%= @user.id %>/mod/privileges">privileges</a>
               <a href="/warning/log/<%= @user.id %>">warnings and suspensions sent to user</a>
               <a href="/warning/new/<%= @user.id %>">warn or suspend user</a>
             </div>
             <div class="h-m-t-6">
               <a href="<%= mod_user_path(@user) %>" class="button is-filled">Show all tools</a>
             </div>
           </div>
-        </div>
-      <% end %>
-      <% if current_user&.id == @user.id %>
-        <div>
+        <% end %>
+        <% if current_user&.id == @user.id %>
           <%= link_to qr_login_code_path, class: 'button is-outlined is-small' do %>
             <i class="fas fa-mobile-alt"></i> Mobile Sign In
           <% end %>
+        <% end %>
+      </div>
+
+      <h2 class="user-profile-heading">Posts</h2>
+      <% if @user.posts.size == 0 %>
+        <p><%= @user.username %> hasn't posted anything yet.</p>
+      <% else %>
+        <div class="item-list user-profile-post-list">
+          <% @user.posts.undeleted.order(score: :desc).first(15).each do |a| %>
+            <%= render 'posts/type_agnostic', post: a, show_type_tag: true, show_category_tag: true %>
+          <% end %>
         </div>
+        <%= link_to user_posts_path(@user), class: "button is-muted" do %>
+          See all &raquo;
+        <% end %>
       <% end %>
     </div>
   </div>
-</div>
+  <div class="grid--cell is-3-lg is-12">
+    <div class="widget is-tertiary">
+      <div class="widget--body user-profile--image">
+        <img alt="user avatar" src="<%= avatar_url(@user, 512) %>" />
+      </div>
+      <% if @user.staff? %>
+      <div class="widget--body h-ta-center" title="This user is an official Codidact Staff member.">
```

Comment:
I'd leave this (and the next line) as simply "staff member", rather than "Codidact staff member", to make it easier for anyone who wants to use the software for themselves.